tests/components/sensibo/test_coordinator.py

```diff
@@ -0,0 +1,74 @@
+"""The test for the sensibo coordinator."""
+from __future__ import annotations
+
+from datetime import timedelta
+from unittest.mock import patch
+
+from pysensibo.exceptions import AuthenticationError, SensiboError
+from pysensibo.model import SensiboData
+import pytest
+
+from homeassistant.components.sensibo.const import DOMAIN
+from homeassistant.components.sensibo.coordinator import SensiboDataUpdateCoordinator
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers.update_coordinator import UpdateFailed
+from homeassistant.util import dt
+
+from .response import DATA_FROM_API
+
+from tests.common import async_fire_time_changed
+
+
+async def test_coordinator(hass: HomeAssistant, load_int: ConfigEntry) -> None:
+    """Test the Sensibo coordinator."""
+
+    coordinator: SensiboDataUpdateCoordinator = hass.data[DOMAIN][load_int.entry_id]
+    assert coordinator.data.parsed["ABC999111"].state == "heat"
+
```

Comment:
We should drop this one as well

Reply:
Should we as it's used to assert the data coming out from the coordinator?

Reply:
We should test the test entities the coordinator are feeding data to instead.  We don't want to test the internals of the integration since it will make it harder to refactor later if we make changes to how the coordinator works (as we have done before)

Reply:
I looked at a few others in the code base and have changed the tests for the coordinator now. It also then matches with your comments.